build.zig

```diff
@@ -8,7 +8,6 @@ pub fn build(b: *std.Build) !void {
     builder_instance = b;
```

Comment:
Didn't realize we had this, we should get rid of it and require the user pass in `b: *std.Build` parameter everywhere. Do we even need `pub fn getJSPath() []u8` in this file anymore?

Reply:
We don't need getJsPath() anymore